theano/gpuarray/basic_ops.py

```diff
@@ -466,12 +481,14 @@ class CGpuKernelBase(COp, GpuKernelBase):
 
     kernel_re = re.compile(r'^#kernel ([a-zA-Z_].*?)$', re.MULTILINE)
 
-    c_support_code = forward_string_meth('c_support_code')
     c_support_code_apply = forward_string_meth('c_support_code_apply')
     c_support_code_struct = forward_string_meth('c_support_code_struct')
     c_init_code_struct = forward_string_meth('c_init_code_struct')
     c_cleanup_code_struct = forward_string_meth('c_cleanup_code_struct')
 
+    def c_code_cache_version_apply(self, node):
```

Comment:
Because if I swap it affects other methods and I don't want to spend time to make sure I don't break anything.